pkg/apis/settings/suite_test.go

```diff
@@ -71,6 +71,15 @@ var _ = Describe("Validation", func() {
 		_, err := (&settings.Settings{}).Inject(ctx, cm)
 		Expect(err).To(HaveOccurred())
 	})
+	It("should fail validation when batchMaxDuration is less then 1s", func() {
+		cm := &v1.ConfigMap{
+			Data: map[string]string{
+				"batchMaxDuration": "0.8s",
```

Comment:
Can you use `ms` just so we know that we're not running into any odd parsing issues with decimals? 
```suggestion
				"batchMaxDuration": "800ms",
```